docs/studio.rst

```diff
@@ -0,0 +1,94 @@
+Studio
+===============
+
+FEDn Studio is a web-based tool for managing and monitoring federated learning experiments. It provides a user-friendly interface for monitoring the progress of training and visualizing the results.
```

Comment:
Maybe add / harmonize Studio intro with features mentioned in the new main README.

Reply:
changed